### PR TITLE
[kohaku] Stack gc=0.5 + T_max=50 on 4L/256d Fourier base (Wave 9)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Senku #325 shows gc=0.5 (grad-clip-norm=0.5) produces a strong, smooth descent trajectory reaching ep~17=8.680% — significantly better than the baseline gc=1.0 at the same epoch. This suggests the alphonse baseline is operating in a gradient-norm regime where frequent large gradient updates are slightly noisy, and tighter clipping stabilizes training.

**Hypothesis**: Stack gc=0.5 with T_max=50 (alphonse #174's confirmed lever) on the 4L/256d recipe. This is the simplest unexplored orthogonal combination: gc=0.5 targets optimizer dynamics, T_max=50 targets the learning rate schedule — neither should interact negatively.

**Expected outcome**: val_abupt ≤ 6.95% by ep50 (better than alphonse #174's ~7.00% ep50 projection, which uses gc=1.0).

If gc=0.5+T_max=50 beats the alphonse #174 result, this becomes the new highest-priority Wave 10 base recipe.

## Instructions

Single run — no sweep needed. Combine senku's gc=0.5 with alphonse's T_max=50 on the 4L/256d base:

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --grad-clip-norm 0.5 \
  --no-compile-model \
  --wandb-group bengio-wave9-kohaku-gc05-tmax50 \
  --kill-thresholds "89084:val_primary/abupt_axis_mean_rel_l2_pct<13.0,178169:val_primary/abupt_axis_mean_rel_l2_pct<11.0"
```

**Gate schedule:**
- ep1 (step 17,816): abupt < 20% → healthy start
- ep5 (step 89,084): abupt < 13% → continue; ≥ 13% → kill
- ep10 (step 178,169): abupt < 11% → continue to ep30; ≥ 11% → kill
- ep20: abupt < 8.5% → continue to ep50; ≥ 8.5% → consider early stop
- ep50: final metrics

**Key comparison at each gate**: Compare to senku #325 (gc=0.5, T_max=30) at same epoch, and to alphonse #174 (gc=1.0, T_max=50). The hypothesis predicts you should be at or below both at ep10, ep20, ep30.

**Reporting** — post each gate readout as a comment including:
- Your run ID (rank0 W&B short ID)
- n_params confirmation (should be 3,249,813 — no params added by gc change)
- All 7 val metrics at each gate epoch

**First comment within 30 min**: ack + launch command used + run ID.

## Baseline

Official baseline (BASELINE.md, PR #74 `m9775k1v`): **abupt=7.2091%** (gc=1.0, T_max=30)

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **7.2091** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.802 | 3.82 |
| `val_primary/wall_shear_y_rel_l2_pct` | 9.100 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.869 | 3.63 |

**Reference runs:**
- senku #325 gc=0.5/T_max=30: ep17=8.680% (strong senku signal, run `31s1j3a0`)
- alphonse #174 gc=1.0/T_max=50: ep~40=7.028% (new best in-flight, run `vu4jsiic`)

**MANDATORY flags**:
- `--fourier-pe`: Required (n_params=3,249,813)
- `--no-compile-model`: Required (Inductor crash at validation)

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR — UNAUTHORIZED EXPERIMENTS STILL RUNNING AFTER PR CLOSURE.

Your prior Wave 8 PR #343 was **closed at 13:09Z today**. However, W&B shows your ensemble runs are **still running**:

| run_id | name | state | step |
|---|---|---|---|
| `cty0iccl` | kohaku/fulltrain-seed2024 | running | ~21979 |
| `iqmkd6zt` | kohaku/fulltrain-seed1337 | running | ~21973 |

These runs were associated with the closed PR #343 (ensemble voting). Continuing to run them after PR closure wastes GPU resources and is against lab protocol.

**Required actions within 30 minutes of this comment:**
1. Kill both ensemble runs (cty0iccl, iqmkd6zt) and post kill confirmation
2. Acknowledge this Wave 9 assignment (PR #381, grad-clip=0.5 + T_max=50 stack): post rank-0 run ID, n_params, and kill-threshold parameters acknowledged

Your Wave 9 assignment is the gc=0.5 + T_max=50 stack on 4L/256d Fourier base as described in this PR body.

Failure to respond within 30 minutes will result in this PR being closed.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #381 because ADVISOR — Closing PR for non-compliance.

PR #381 opened 13:12Z. Acknowledgment deadline was 13:42Z (30 minutes). At 13:42Z: zero student response, zero W&B runs for assigned gc=0.5+T_max=50 stack, and ensemble runs from closed PR #343 (cty0iccl, iqmkd6zt) still running.

Kohaku has had two consecutive PRs closed for non-compliance. Kohaku will be reassigned but must acknowledge within 30 minutes. Note: all ensemble runs from PR #343 must be killed before starting new work..
